assets/stylesheets/_components.scss

```diff
@@ -370,3 +370,12 @@
 @import 'signup/validation-fieldset/style';
 @import 'support/support-user/style';
 @import 'vip/vip-logs/style';
+
+// ==========================================================================
```

Comment:
>Hmm that is a great question.  Seems fine to me to place the import here, but have to agree grabbing from `node_modules` looks/feels strange.  The [file](https://github.com/bvaughn/react-virtualized/blob/master/styles.css) isn't huge, but we could take the approach done with `component/tip` and grab the base styles and place within calypso itself [#](https://github.com/Automattic/wp-calypso/blob/dabf92fa84865576ecd8d806bd21c9cc4184e73d/client/components/popover/style.scss)

Yeah, not a straight-forward answer here. The issue with copying the styles is more a maintainability concern, requiring us to keep on top of syncing the styles any time we choose to upgrade `react-virtualized`. As strange as it looks to reach into `node_modules`, I can't think of a case where it wouldn't work as we'd expect, especially since [`node_modules` Makefile target is a prerequisite to our style build](https://github.com/Automattic/wp-calypso/blob/3901c935163c986626ca7c73e0ad2bdc72350ecf/Makefile#L129).